components/brave_new_tab_ui/containers/newTab/settings/icons/sponsoredImageInfo.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+export function SponsoredImageInfoIcon () {
```

Comment:
Semi-related question: If we had a lot of components using this pattern in our codebase, does it negatively affect performance for sessions which don't result in uses of those components. Instead of exporting functions that don't get called, there would instead be a lot of React.createClass calls during initial JS module parsing. A single (or a few hundred??) instances of this of course won't matter, but it would be interesting to know if this is a best practice and at what point it causes negative impact.